### ISSUE TITLE
Form with error if I use an user type editor 

### ISSUE THREAD

**Issue by Bulbow** [yikesinc/yikes-inc-easy-mailchimp-extender#442]
Dear YIKES,

I created and configurated your Easy Forms for MailChimp by YIKES but when I login with profile editor the homepage show me this error...

Notice: Undefined variable: admin_class in C:\inetpub\wwwroot\ban\wp-content\plugins\yikes-inc-easy-mailchimp-extender\public\partials\shortcodes\process_form_shortcode.php on line 1123

if I use an user with profile admin the form its ok!

Please, can you help me?
Best Regards,
Marcos

![error_editor1](https://cloud.githubusercontent.com/assets/17203872/13016201/6d0649c2-d1a4-11e5-8ed2-9b53af938a97.JPG)
![error_editor2](https://cloud.githubusercontent.com/assets/17203872/13016202/6d09bfa8-d1a4-11e5-928a-75224d6cf430.JPG)

**Comment by EvanHerman** [comment]
Thanks Marcos,

I'll take a look into this issue.


Evan

**Comment by EvanHerman** [comment]
I see the issue, and will have this issue patched up in 6.0.4.1 release coming out in the coming days.

In the meantime, you should be able to use the following snippet to resolve the issue:

```php
function alter_yikes_mailchimp_caps( $capability ) {
     return 'edit_posts';
}
add_filter( 'yikes-mailchimp-user-role-access', 'alter_yikes_mailchimp_caps' );
```

The snippet above will allow users with the editor role and above to view and click the 'edit form' link below the form.

In the next release, this issue will be resolved.


Thank you again for reporting the issue here!


Evan
